neo/rawio/neuralynxrawio/ncssections.py

```diff
@@ -1,3 +1,38 @@
+"""
+Objects to represent continguous sections of samples in a Neuralynx continuously sampled .Ncs
+file. These sections are considered contiguous in that they have a start time, a sampling rate,
+and a length.
+
+Defining these sections is complicated due to the physical structure of .Ncs files which contain
+both a header and a set of fixed length records, CscRecords.
+
+Each CscRecord has a start time in microseconds, a channel id (chan_id), a stating sampling
```

Comment:
```suggestion
Each CscRecord has a start time in microseconds, a channel id (chan_id), a stated sampling
```

Is this what you meant?